cloud/store/bootstrap.go

```diff
@@ -106,6 +112,17 @@ func (b *Bootstrapper) notify(ctx context.Context, servers []string) (err error)
 	return
 }
 
+// servers retrieves a list of server addresses based on their names and ports.
+func (b *Bootstrapper) servers(candidates []string, serverPortMap map[string]int) []string {
+	candidates = candidates[:0]
```

Comment:
**suggestion:** may be we use `clear(candidates)` basically it's the same but could improve readability

Reply:
I favor this approach over creating a new function solely for returning a slice. 
The `clear(candidate)` function does not actually clear anything; it simply returns a new slice. Therefore, slicing is makes more sense to me

Reply:
i meant `clear` the builtin function